File: c/9.c
/*
    main 함수 안의 내용만 수정해주세요.
    수정할 경우, 자동 채점 프로그램이 제대로 동작하지 않을 가능성이 있습니다.
*/

#include <stdio.h>

int main() {
    // 이곳에 코드를 작성해주세요!
{
    int i,n,sum=0;
    printf("1부터n까지의 합 계산\n");
    printf("정수n입력:");
    scanf("%d",&n);
    for(i=1;i<=n;i+=1)
      sum+=i;
    printf("1부터 %d까지의 합:%d\n",n,sum);
}
    return 0;
}

File: python/9.py
"""
    main 함수 안의 내용만 수정해주세요.
    수정할 경우, 자동 채점 프로그램이 제대로 동작하지 않을 가능성이 있습니다.
"""

def main():
    while True:
        n = int(input("양의 정수 n을 입력하세요: "))
        
        if n > 0:
            break
        else:
            print("양의 정수를 입력해야 합니다. 다시 입력하세요.")

    sum_of_integers = sum(range(1, n + 1))
    print(sum_of_integers)

    # 이곳에 코드를 작성해주세요!



if __name__ == '__main__':
    main()
